manifests/params.pp

```diff
@@ -46,8 +46,11 @@
       $config_group       = 'root'
       $java_dir           = '/usr/share/collectd/java'
       $default_python_dir = '/usr/local/lib/python2.7/dist-packages'
-      $manage_repo        = true
       $package_configs    = {}
+      $manage_repo        = $facts['os']['release']['full'] ? {
```

Comment:
Perhaps shorter to use `$manage_repo = $facts['os']['release']['full'] != '18.04'`